pyproject.toml

```diff
@@ -22,31 +22,35 @@ pydot = "^1.4.1"
 [tool.poetry.dev-dependencies]
 codecov = "^2.0.15"
 cython = "^0.29.14"
-gensim = "^3.8.1"
 jupyter = "^1.0.0"
 jupyterlab = "^1.2.4"
-keras = "^2.3.1"
 matplotlib = "^3.1.2"
```

Comment:
This one, I'm not even sure why we need it at all :sweat_smile:
I'll have to check where it's being used.